### PR TITLE
Wizard recipe: Pangomm-v2.49.1

### PR DESCRIPTION
**Pull request by pepijndevos**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: Pangomm
* Version: v2.49.1

@staticfloat please review and merge.

### CODE REVIEW

**Comment by pepijndevos** [comment]
```
/usr/lib/libexpat.so.1: undefined reference to `getrandom'

Several platforms failed to build:
 - Linux x86_64 {libc=musl}
```

**Comment by pepijndevos** [comment]
Another thought: since you seem to usually want to just have `supported_platforms` so you can see the errors, maybe BinaryBuilder wizard should offer that option.

**Comment by pepijndevos** [comment]
wat

```
[ Info: Found a valid dl path libpangomm-2.48.1.dylib while looking for libpangomm-2.48
[ Info: Found a valid dl path libpangomm-2.48.dylib while looking for libpangomm-2.48

[ Info: Could not locate libpangomm-2.48 inside
┌ Error: Built Pangomm but pangomm still unsatisfied:
```

**Comment by pepijndevos** [comment]
armv6: `Run-time dependency pangocairo found: NO (tried pkgconfig and cmake)`

File: P/Pangomm/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = expand_cxxstring_abis(supported_platforms(; experimental=true))
+
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct(["libpangomm-$(version.major)", "libpangomm-$(version.major).$(version.minor)"], :libpangomm)
```
Comment (giordano) [inline P/Pangomm/build_tarballs.jl:34]: The version of the package is 2.49 but the library has name 2.48.  Sigh
```suggestion
    LibraryProduct(["libpangomm-$(version.major)", "libpangomm-$(version.major).48"], :libpangomm)
```

**Comment by giordano** [comment]
> ```
> /usr/lib/libexpat.so.1: undefined reference to `getrandom'
> 
> Several platforms failed to build:
>  - Linux x86_64 {libc=musl}
> ```
> 

When target == host, some tools pick up libraries from the system.  I deleted it to avoid confusion.

> wat
> 
> ```
> [ Info: Found a valid dl path libpangomm-2.48.1.dylib while looking for libpangomm-2.48
> [ Info: Found a valid dl path libpangomm-2.48.dylib while looking for libpangomm-2.48
> 
> [ Info: Could not locate libpangomm-2.48 inside
> ┌ Error: Built Pangomm but pangomm still unsatisfied:
> ```
> 

Trick to find the expected names (this is also [documented](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.LibraryProduct)): 

```julia
julia> using Base.BinaryPlatforms

julia> BinaryPlatforms.parse_dl_name_version("libpangomm-2.48.1.dylib", "macos")[1]
"libpangomm-2"

julia> BinaryPlatforms.parse_dl_name_version("libpangomm-2.48.so", "linux")[1]
"libpangomm-2.48"
```

But wouldn't it be awesome if these libraries just followed standard soversioning schemes?  They even got the version wrong, it should have been 2.49, not 2.48 :roll_eyes: 

>     armv6: Run-time dependency pangocairo found: NO (tried pkgconfig and cmake)

Correct, we don't have Pango for that platform

**Pull request merged**